browser-test/playwright.Dockerfile

```diff
@@ -1,28 +1,50 @@
 # syntax=docker/dockerfile:1
-FROM mcr.microsoft.com/playwright:v1.39.0-jammy
+FROM ubuntu:22.04
```

Comment:
question (blocking): what is the process for managing our playwright version after this change?

Reply:
It should pull from the package(-lock).json version when `npm install` is called

Reply:
For example there is a newer playwright release (v1.40.0) which came out last Thursday. We still have our package.json set to v1.39.0.

You can `docker exec` into a running image and run `npm list playwright` and it shows v1.39.0. If you then update the package.json to v1.40.0 and build the image if uses that version. 

The change in the package.json should trigger the github action to build and push the updated browser tests.

Reply:
got it, thank you